qutip/core/operators.py

```diff
@@ -64,6 +65,10 @@ def jmat(j, which=None):
         Which operator to return 'x','y','z','+','-'.
         If no args given, then output is ['x','y','z']
 
+    dtype : qutip.data.Data or str
+        Storage representation. Any data-layer known to `qutip.data.to` is
+        accepted. Default `qutip.data.Dense` (numpy.ndarray like).
```

Comment:
The idea of the default here is one thing that we'll probably revisit (I envisage being able to define a set of Dispatcher rules on a per-function basis in the exact same way you do your global settings), but I'm entirely happy to have it work like this to get the ball rolling.